openslides/motions/pdf.py

```diff
@@ -278,15 +278,18 @@ def motion_poll_to_pdf(pdf, poll):
 
     # set number of ballot papers
     if ballot_papers_selection == "NUMBER_OF_DELEGATES":
-        # TODO: get this number from users
         try:
-            if Group.objects.get(pk=3):  # TODO: Find a better way
-                number = User.objects.filter(groups__pk=3).count()
-        except Group.DoesNotExist:
+            from openslides.users.models import Group
```

Comment:
This will only fail when the user app is deleted, but not, if another user app is used in the settings.